cache.go

```diff
@@ -128,6 +128,7 @@ func (c *cache) Get(k string) (interface{}, bool) {
 	if item.Expiration > 0 {
 		if time.Now().UnixNano() > item.Expiration {
 			c.mu.RUnlock()
+			c.Delete(k)
```

Comment:
What about adding this change also in the ```GetWithExpiration```  https://github.com/patrickmn/go-cache/pull/75/files#diff-56ffa6659747ba7a23d2b5597a4cda5dR143 ?

Reply:
done

Reply:
I think you'll only want to call `Delete` _if_ the cache has an eviction function.